### PR TITLE
EDU-766 Support writing comments

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-766

Screen Shot 2022-09-01 at 17 00 05
Screen Shot 2022-09-01 at 17 00 15
Screen Shot 2022-09-01 at 17 00 36
Screen Shot 2022-09-01 at 17 00 46

### CODE REVIEW

**Comment by ahelmberger** [comment]
Looks really nice! There's a problem though, in case a document doesn't have any comments yet, the accordion panel for the new topic cannot be expanded (thus preventing the user from writing the first comment).

File: src/components/pdf-document.js [hunk 1]
```diff
@@ -17,16 +17,26 @@ function PdfDocument({ file, pageNumber, stretchDirection, showTextOverlay, onLo
...
     // In order to not have flickering on the page while the new page renders we keep the dimensions
     // of the old page fixed until `handlePageRenderSuccess` has been called by the next page,
     // or any of the error components are rendered.
     const boundingClientRect = viewerRef.current.getBoundingClientRect();
     setViewerStyle({ height: boundingClientRect.height, width: boundingClientRect.width });
     setActualPageNumber(pageNumber);
+
+    return () => {
+      isMounted.current = false;
+    };
```
Comment (ahelmberger) [inline src/components/pdf-document.js:39]: Cool submarine fix :smiling_imp:

File: src/components/pages/doc.js [hunk 2]
```diff
@@ -44,11 +42,10 @@ import {
   confirmSectionDelete,
   confirmSectionHardDelete
 } from '../confirmation-dialogs.js';
+import CommentsPanel from '../comments-panel.js';
```
Comment (ahelmberger) [inline src/components/pages/doc.js:45]: This line is rather short :smirk:

Comment (adrianaluchian) [reply src/components/pages/doc.js]: It went up to its short friends 😅

File: src/components/pdf-document.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pdf-document.js:39]: 😂 It was time.

File: src/components/comments-panel.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+addCommentButtonText:
+  en: Write comment
+  de: Kommentar schreiben
+postCommentButtonText:
+  en: Post comment
+  de: Kommentar aufgeben
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/components/comments-panel.yml]: Got it, changed, thank you!

**Comment by adrianaluchian** [comment]
> Looks really nice! There's a problem though, in case a document doesn't have any comments yet, the accordion panel for the new topic cannot be expanded (thus preventing the user from writing the first comment).

Good catch! On it..

**Comment by adrianaluchian** [comment]
> > Looks really nice! There's a problem though, in case a document doesn't have any comments yet, the accordion panel for the new topic cannot be expanded (thus preventing the user from writing the first comment).
> 
> Good catch! On it..

It was a strange rendering glitch due to the ant design `Panel` not being able to render its own `children` while itself being rendered through our `Restricted` component. It's fixed now.

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**